lib_expense/src/test/java/com/example/expenselogger/lib_expense/executor/TestAsyncExecutor.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.expenselogger.lib_expense.executor
+
+import com.example.expenselogger.executor.threading.AsyncExecutor
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+
+public class TestAsyncExecutor : AsyncExecutor {
```

Comment:
Nice idea adding this. However I think it should live in the executor module, in the test directory. Then you can add the executor module to the lib-search-module using `testImplementation`. This TestDispatcher will still be required by other modules eventually.

Reply:
Okay, I'll do that